src/microbe_stage/MicrobeStage.tscn

```diff
@@ -856,7 +851,6 @@ margin_bottom = 12.5
 grow_horizontal = 0
 rect_min_size = Vector2( 55, 25 )
 custom_fonts/font = SubResource( 15 )
-text = "0%"
```

Comment:
Aren't a lot of these texts needed?
Should there be a filter or something in the extract process (or a ruby based .pot post process step) to let these be here?